CHANGELOG.md

```diff
@@ -42,6 +42,11 @@ All notable changes to the Imperative package will be documented in this file.
 ## `5.0.0-next.202009251501`
 
 - Enhancement: add support for CLIs that want to run as a persistent process (daemon mode).
+## `4.13.0`
```

Comment:
Could you add a blank line above this header? Sorry for being picky 😢

Reply:
`/facepalm` 🤦 
Will do!